README.md

```diff
@@ -1,2 +1,18 @@
-# sfdc-carshop
-Car Accessories and Spare Parts Selling Application for Salesforce
+# Salesforce DX Project: Next Steps
```

Comment:
It is ok to keep the info added by SFDX tool, but since this is the README of the project itself, it would be good to set main title with something like "Saleforce Car Shop", a brief description of the app and then, the SFDX steps. Furthermore, SFDX steps might not be required at this point, so optionally, I think those could be removed